pkg/kubelet/kubelet.go

```diff
@@ -1722,12 +1752,41 @@ func (kl *Kubelet) makePodDataDirs(pod *api.Pod) error {
 	return nil
 }
 
+// syncPod is the transaction script for the sync of a single pod.
+//
+// Arguments:
+//
+// pod - the pod to sync
+// mirrorPod - the mirror pod for the pod to sync, if it is a static pod
+// podStatus - the current status (TODO: always from the status manager?)
+// updateType - the type of update (CREATE,ADD,DELETE,RECONCILE)
```

Comment:
ADD, UPDATE, REMOVE, ... are all things we stick in a kubetypes.PodUpdate channel which is not what this operation takes..

syncPod takes a kubetypes.SyncPodType, which is just create, update, and sync.

now its worth asking why we have two similar but different types.